config/services.yaml

```diff
@@ -75,6 +75,8 @@ services:
 
     Github\Api\Issue\Labels:
         factory: ['@Github\Api\Issue', labels]
+        calls:
+            - ['setPerPage', [100]]
```

Comment:
We're telling the `Labels` API to return 100 results but we're *also* using `ResultPager` to fetch all the results? That seems potentially redundant - are we doing it just to be safe?

Reply:
We currently have 87 labels. If we add 14 more this will be a silent error. So yes, the ResultPager is not really needed at the moment.

Reply:
Btw, The ResultPager is just making extra requests for us. 

If we left this as (default) 20, the Result pager would make 5 request.